x-pack/plugins/cases/public/components/custom_fields/flyout.test.tsx

```diff
@@ -111,6 +111,7 @@ describe('CustomFieldFlyout ', () => {
       'value',
       customFieldsConfigurationMock[0].label
     );
+    expect(await screen.findByTestId('custom-field-type-selector')).toHaveAttribute('disabled');
```

Comment:
Can you add the same check in `x-pack/plugins/cases/public/components/custom_fields/form.test.tsx`? Also one for not being disabled if the initial value is not set.